src/main/modules/i18n/index.ts

```diff
@@ -8,12 +7,12 @@ export class LanguageToggle {
   private supportedLanguages = ['en', 'cy'];
 
   public enableFor(app: Express): void {
-    app.use((req: ReqWithSession, res: Response, next: NextFunction) => {
+    app.use((req, res, next) => {
       if (req.method === 'GET' && req.query['lng']) {
         const requestedLanguage = req.query['lng'] as string;
 
         if (this.supportedLanguages.includes(requestedLanguage)) {
-          req.session.lang = requestedLanguage;
+          req.session['lang'] = requestedLanguage;
```

Comment:
probably related to removing types or is this a preferred way?

Reply:
`req.query` is used in the same way above (`req.query['lng']`). So `req.session['lang']` is used to match the type from `app: Express`.